src/systemtest/java/com/ericsson/ei/systemtests/TestSteps.java

```diff
@@ -0,0 +1,30 @@
+package com.ericsson.ei.systemtests;
```

Comment:
Why are we adding a new test steps file that does nothing? I guess you are suppose to test that you can access the variables you set somewhere?

Reply:
Yes.... it is to demo working test.... I delete in the next commit